packages/loctool/test/webOSXliff.test.js

```diff
@@ -0,0 +1,811 @@
+    });
+    test("webOSXliffGetResourcesMultiple", function() {
+        expect.assertions(11);
+        var x = new webOSXliff();
+        expect(x).toBeTruthy();
+
+        var res = new ResourceString({
+            source: "Asdf asdf",
+            sourceLocale: "en-KR",
+            key: "foobar",
+            pathName: "foo/bar/asdf.js",
+            project: "webapp",
+            origin: "source"
+        });
+
+        x.addResource(res);
+        res = new ResourceString({
+            source: "baby baby",
+            sourceLocale: "en-KR",
+            key: "huzzah",
+            pathName: "foo/bar/j.js",
+            project: "webapp",
+            origin: "origin"
+        });
+
+        x.addResource(res);
+
+        var reslist = x.getResources({
+            sourceLocale: "en-KR"
+        });
+
+        expect(reslist).toBeTruthy();
+        expect(reslist.length).toBe(2);
+
+        expect(reslist[0].getSource()).toBe("Asdf asdf");
+        expect(reslist[0].getSourceLocale()).toBe("en-KR");
+        expect(reslist[0].getKey()).toBe("foobar");
+        expect(reslist[0].getPath()).toBe("foo/bar/asdf.js");
+
+        expect(reslist[1].getSource()).toBe("baby baby");
+        expect(reslist[1].getSourceLocale()).toBe("en-KR");
+        expect(reslist[1].getKey()).toBe("huzzah");
+        expect(reslist[1].getPath()).toBe("foo/bar/j.js");
+    });
+    test("webOSXliffDeserialize", function() {
+        expect.assertions(8);
+        var x = new webOSXliff();
+        x.deserialize(
+        '<?xml version="1.0" encoding="utf-8"?>\n' +
+        '<xliff version="2.0" srcLang="en-KR" trgLang="de-DE" xmlns:l="http://ilib-js.com/loctool">\n' +
+        '  <file id="sample-webos-cs_f1" original="sample-webos-c">\n' +
+        '      <group id="sample-webos-c_g1" name="c">\n' +
+        '        <unit id="1">\n' +
+        '          <segment>\n' +
+        '            <source>Asdf asdf</source>\n' +
+        '            <target>foobarfoo</target>\n' +
+        '          </segment>\n' +
+        '        </unit>\n' +
+        '      </group>\n' +
+        '  </file>\n' +
+        '</xliff>');
+
+        expect(x).toBeTruthy();
+
+        var result = x.getResources();
+        expect(result).toBeTruthy();
+        expect(result.length).toBe(1);
+
+        expect(result[0].getSource()).toBe("Asdf asdf");
+        expect(result[0].getSourceLocale()).toBe("en-KR");
+        expect(result[0].getTargetLocale()).toBe("de-DE");
+        expect(result[0].getKey()).toBe("Asdf asdf");
+        expect(result[0].getTarget()).toBe("foobarfoo");
+    });
+    test("webOSXliffDeserialize_metadata", function() {
+        expect.assertions(8);
+
+        var x = new webOSXliff({
+            metadata: {"device-type": "SoundBar"}
+        });
+        x.deserialize(
+        '<?xml version="1.0" encoding="utf-8"?>\n' +
+        '<xliff version="2.0" srcLang="en-KR" trgLang="ko-KR" xmlns:l="http://ilib-js.com/loctool">\n' +
```

Comment:
Please add the namespace to ensure proper parsing and validation of metadata elements.
ex) 
<xliff xmlns="urn:oasis:names:tc:xliff:document:2.0" version="2.0"
       xmlns:mda="urn:oasis:names:tc:xliff:metadata:2.0"
       srcLang="en-KR" trgLang="ko-KR">

Reply:
I updated it.